src/SearchUsers/SearchUsers.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+	Box,
+	Button,
+	Container,
+	Grid,
+	TextareaAutosize,
+	TextField,
+} from '@material-ui/core'
+import axios from 'axios'
+import React, { FC, useState } from 'react'
+import { ConfigPage } from '../ConfigPage/ConfigPage'
+import { ApiService } from '../services/ApiService'
+import './SearchUsers.css'
+
+interface SearchUsersProps {
+	api: ApiService
+	defaultSearchValue?: string
+}
+
+const SearchUsers: FC<SearchUsersProps> = ({
+	api,
+	defaultSearchValue = '',
+}) => {
+	const [newSearchValue, setNewSearchValue] = useState(defaultSearchValue)
+	const [searchUserName, setSearchUserNameValue] = useState('')
+	const [searchUserLevel, setSearchUserLevelValue] = useState('')
+	const [searchUserIcon, setSearchUserIconValue] = useState('')
```

Comment:
I might name these a little differently - perhaps something like `resultName`, `resultLevel`, and `resultIcon` since we know we're searching users and technically these vars deal w/ the result of the search, not the search itself

```suggestion
	const [searchUserName, setSearchUserNameValue] = useState('')
	const [searchUserLevel, setSearchUserLevelValue] = useState('')
	const [searchUserIcon, setSearchUserIconValue] = useState('')
```